my.css

```diff
@@ -352,3 +352,161 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  max-width: 1150px;
+  width: 100%;
+  margin: 0 auto;
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 55%;
+  height: 844px;
+  
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 340px;
+  height: 370px;
+  padding-left: 90px;
+
+}
+.bags-img1 {
+  position: absolute;
+  width: 540px;
+  height: 740px;
+  left: 160px;
+}
+
+.bagclach__iner {
+  width: 45%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+
+}
+.card {
+  width: 350px;
+  padding: 40px 55px 45px 65px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)